code/iaas/storage-service/src/main/java/io/cattle/platform/storage/api/filter/ExternalTemplateInstanceFilter.java

```diff
@@ -10,31 +11,58 @@
 import io.github.ibuildthecloud.gdapi.request.resource.AbstractResourceManagerFilter;
 import io.github.ibuildthecloud.gdapi.request.resource.ResourceManager;
 import io.github.ibuildthecloud.gdapi.validation.ValidationErrorCodes;
+import io.cattle.platform.docker.client.DockerImage;
 
 import java.util.List;
 import java.util.Map;
 
 import javax.inject.Inject;
 
+import com.netflix.config.DynamicStringProperty;
+
 public class ExternalTemplateInstanceFilter extends AbstractResourceManagerFilter {
 
     SchemaFactory schemaFactory;
     @Inject
     StorageService storageService;
 
+    public static final DynamicStringProperty DEFAULT_REGISTRY = ArchaiusUtil.getString("default.registry");
```

Comment:
Nothing ever does anything to use this or perform the default registry functionality?